service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.switchboard.switchboardapp;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SwitchboardappApplication {
```

Comment:
`SwitchboardappApplication` is repetitive, maybe `SwitchboardApplication`?

Reply:
Fixed.